qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -369,9 +368,8 @@ def _qctrl_expanded_export_content(self, file_type, coordinates):
         control_info = None
         amplitude_x = self.amplitude_x
         amplitude_y = self.amplitude_y
-        if coordinates == CARTESIAN:
-            if file_type == CSV:
-
+        if coordinates == Coordinates.CYLINDRICAL.value:
+            if file_type == FileType.CSV.value:
```

Comment:
I guess this was a problem before too, but could you update the docstring of this function to capitalise "CSV" and "JSON"?

Reply:
done